integration/framework.go

```diff
@@ -37,11 +40,23 @@ type kubeFramework interface {
 	// Kube client
 	Client() *kube_client.Client
 
-	// Run kubectl.sh command
-	RunKubectlCmd(cmd ...string) (string, error)
+	// Parses and Returns a replication Controller object contained in 'filePath'
+	ParseRc(filePath string) (*api.ReplicationController, error)
 
-	// Run kubectl.sh command
-	RunKubecfgCmd(cmd ...string) (string, error)
+	// Parses and Returns a service object contained in 'filePath'
+	ParseService(filePath string) (*api.Service, error)
+
+	// Creates a kube service.
+	CreateService(ns string, service *api.Service) (*api.Service, error)
```

Comment:
nit: The signature looks unnatural. We don't need to return *api.Service. I think we can hide the k8 api format and do the simpler version of returning just the error. Same for CreateRC.